lib/pbench/server/database/alembic.ini

```diff
@@ -7,8 +7,14 @@ script_location = alembic
 # template used to generate migration files
 # file_template = %%(rev)s_%%(slug)s
 
-# timezone to use when rendering the date
-# within the migration file as well as the filename.
+# sys.path path, will be prepended to sys.path if present.
+# defaults to the current working directory.
+prepend_sys_path = .
+
+# timezone to use when rendering the date within the migration file
+# as well as the filename.
+# If specified, requires the python-dateutil library that can be
+# installed by adding `alembic[tz]` to the pip requirements
```

Comment:
Do you want to join lines 15 and 16 together (and re-wrap)?

Reply:
I was hoping not to stay away from further changes to this autogenerated file.  I think all we need to change is that `sqlalchemy.url` line in the file.